README.md

```diff
@@ -1,36 +1,44 @@
-# DenoDB database storage adapter for grammY
+# DenoDB Database Storage Adapter for grammY
 
 Database storage adapter that can be used to [store your session data](https://grammy.dev/plugins/session.html) via [DenoDB](https://github.com/eveningkid/denodb) when using sessions.
 
 ## Instructions
 
 1. Import the adapter
-```ts
-import { DenoDBAdapter } from "https://deno.land/x/zumoshi/grammy_session_denodedb/mod.ts";
-```
+
+   ```ts
+   import { DenoDBAdapter } from "https://deno.land/x/zumoshi/grammy_session_denodb/mod.ts";
```

Comment:
Will need to be adjusted before publish. There is no such thing as orgs on `deno.land/x` so it will have to be `https://deno.land/x/grammy_session_denodb/mod.ts`.